engine/table/src/main/java/io/deephaven/engine/table/impl/replay/Replayer.java

```diff
@@ -190,8 +190,8 @@ public Table replay(Table dataSource, String timeColumn) {
         if (dataSource.isRefreshing()) {
             dataSource = dataSource.snapshot();
         }
-        final ReplayTable result =
-                new ReplayTable(dataSource.getRowSet(), dataSource.getColumnSourceMap(), timeColumn, this);
+        final ReplayTable result = new ReplayTable(
+                dataSource.getRowSet(), dataSource.getColumnSourceMap(), timeColumn, this, updateGraph);
         currentTables.add(result);
         if (deltaNanos < Long.MAX_VALUE) {
             updateGraph.addSource(result);
```

Comment:
I don't love the pattern wherein the replayer removes itself from the update graph, but is added externally. Maybe we should add a "start()" method to ReplayTable?